.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,83 @@
+# This workflow aims to automatically rebuild eamxx documentation
```

Comment:
I think this should be removed.  Or it needs to be blocked out with something like "if: ${{ github.event.repository.name = 'scream' }}"

Reply:
Good catch!